app/src/main/res/layout/fragment_settings.xml

```diff
@@ -19,154 +19,154 @@
-                android:gravity="start|center_vertical"/>
+                android:gravity="start|center_vertical" />
 
             <View
                 android:layout_width="match_parent"
                 android:layout_height="0.5dp"
-                android:background="@drawable/line_separator_gray"/>
+                android:background="@drawable/line_separator_gray" />
 
             <TextView
-                android:id="@+id/btnDashcamSettings"
+                android:id="@+id/btnSocialSettings"
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/standard_touch_size"
                 android:paddingEnd="@dimen/margin_padding_size_6"
                 android:paddingStart="@dimen/margin_padding_size_0"
                 app:drawableEndCompat="@drawable/ic_polygon"
-                app:drawableStartCompat="@drawable/ic_triangle_settings_small"
-                android:text="@string/dashcam_settings"
+                app:drawableStartCompat="@drawable/baseline_email_24"
+                android:text="@string/quick_messages"
                 android:textColor="@color/gray3"
                 android:textSize="@dimen/text_size_16"
                 android:clickable="true"
                 android:focusable="true"
                 android:background="?android:attr/selectableItemBackground"
                 android:drawablePadding="@dimen/margin_padding_size_8"
-                android:gravity="start|center_vertical"/>
+                android:gravity="start|center_vertical" />
 
             <View
                 android:layout_width="match_parent"
                 android:layout_height="0.5dp"
-                android:background="@drawable/line_separator_gray"/>
+                android:background="@drawable/line_separator_gray" />
 
             <TextView
-                android:id="@+id/btnHistorySettings"
+                android:id="@+id/btnMapsSettings"
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/standard_touch_size"
                 android:paddingEnd="@dimen/margin_padding_size_6"
                 android:paddingStart="@dimen/margin_padding_size_0"
                 app:drawableEndCompat="@drawable/ic_polygon"
-                app:drawableStartCompat="@drawable/ic_history_settings_small"
-                android:text="@string/history_settings"
+                app:drawableStartCompat="@drawable/ic_maps_settings_small"
+                android:text="@string/point_of_interest"
                 android:textColor="@color/gray3"
                 android:textSize="@dimen/text_size_16"
                 android:clickable="true"
                 android:focusable="true"
                 android:background="?android:attr/selectableItemBackground"
                 android:drawablePadding="@dimen/margin_padding_size_8"
-                android:gravity="start|center_vertical"/>
+                android:gravity="start|center_vertical" />
```

Comment:
We have to discuss where we want to have the change map style options. Depending on our decision we might need to change this element